scripts/weekly-summary

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# This script generates a weekly report of kickstart-test runs.
+# It downloads the daily json summary reports and combines them into a single text
+# report.
+#
+# It requires a github token in GITHUB_TOKEN environmental variable. This token
+# needs to have access to the kickstart-tests artifacts via the github API.
+
+import argparse
+from datetime import datetime, timedelta
+from glob import glob
+import io
+import json
+import os
+import sys
+from subprocess import check_call
+import tempfile
+from urllib.request import Request, urlopen
+import zipfile
+
+import pycurl
```

Comment:
Is this a dependency that needs listing somewhere?

Reply:
Oh good point.